contracts/colonyNetwork/ColonyNetworkDeployer.sol

```diff
@@ -93,6 +94,43 @@ contract ColonyNetworkDeployer is ColonyNetworkStorage {
     return colonyAddress;
   }
 
+  function createColonyForFrontend(
```

Comment:
Why not have two functions, one which accepts a token and one which deploys? The frontend could choose which to call.

Reply:
Mostly I don't want to encourage the idea that to add a new flow we just add a new function. I dislike that we're putting this function in in the first place.